gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -816,15 +824,128 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("SPARK-10634 timestamp written and read as INT64 - truncation")
     .exclude("Migration from INT96 to TIMESTAMP_MICROS timestamp type")
     .exclude("SPARK-10365 timestamp written and read as INT64 - TIMESTAMP_MICROS")
+    // Schema clipping.
+    .exclude("SPARK-10301 requested schema clipping - out of order")
+    .exclude("SPARK-10301 requested schema clipping - requested schema contains physical schema")
+    .exclude("SPARK-10301 requested schema clipping - schemas overlap but don't contain each other")
+    .exclude("SPARK-10301 requested schema clipping - deeply nested struct")
   // requires resource files from Vanilla spark jar
   // enableSuite[GlutenParquetRebaseDatetimeV1Suite]
   // enableSuite[GlutenParquetRebaseDatetimeV2Suite]
   enableSuite[GlutenParquetV1SchemaPruningSuite]
     // spark.sql.parquet.enableVectorizedReader=true not supported
     .excludeByPrefix("Spark vectorized reader - ")
+    // Struct reader does not support implicit schema pruning.
+    .exclude("Non-vectorized reader - without partition data column - " +
```

Comment:
As so many tests need to be excluded, can we just use a white list for it if white list size is much smaller? I.e., just include the workable tests.

Reply:
Although ~30 of them are excluded, there are still ~60 unit tests in this suite, so the list would be smaller with exclude.